lib/tracker_api/client.rb

```diff
@@ -113,7 +116,7 @@ def paginate(path, options = {}, &block)
           if block_given?
             yield(data, @last_response)
           else
-            data.concat(@last_response.body) if @last_response.body.is_a?(Array)
+            data.concat(@last_response.body) if @last_response.body.is_a?(Array) || @last_response.body.is_a?(Hash)
           end
         end
       end
```

Comment:
It seems like `#paginate` is only ever expecting Arrays. I didn't dig in on the repercussions of also allowing Hashes; tests pass.